deltachat-ffi/src/lib.rs

```diff
@@ -1709,10 +1709,14 @@ pub unsafe extern "C" fn dc_imex(
 
     let param1 = to_opt_string_lossy(param1);
 
-    spawn(async move {
-        imex::imex(&ctx, what, param1)
-            .await
-            .log_err(ctx, "IMEX failed")
+    std::thread::spawn(move || {
```

Comment:
This must not manually spawn a new thread, it should use `async_std::task::spawn` as before

Reply:
@dignifiedquire this still produces the compile error: https://github.com/deltachat/deltachat-core-rust/pull/1749/checks?check_run_id=914055708